hppc/src/test/templates/com/carrotsearch/hppc/SortedIterationKTypeVTypeHashMapTest.java

```diff
@@ -78,6 +95,43 @@ public void testIterator() {
     assertThat(iterator.hasNext()).isFalse();
   }
 
+  /*! #if ($TemplateOptions.KTypeGeneric) !*/
```

Comment:
Could we move those methods as a static method naturalOrder() (like JDK Comparator#naturalOrder) in KTypeComparator?

Reply:
There is no generated ObjectComparator (there are interfaces generated for primitive types because JDK doesn't have all the equivalents). Yes, you could move the templated code from the test there.